products/statement-generator/src/flows/BeforeYouBegin.tsx

```diff
@@ -42,7 +47,7 @@ const BeforeYouBegin = ({ onChangeAffirmation }: ComponentProps) => {
         {t('disclaimer.text')}
       </div>
 
-      <div className={`${utilityClasses.buttonContainer} adjacent-mar-top`}>
+      <div className={classes.buttonContainer}>
```

Comment:
Leave this as `<div className={utilityClasses.buttonContainer}>`. Removing the additional className is good here though! 👍